test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/LanguageServiceTests.cs

```diff
@@ -336,6 +337,75 @@ await langService.DoHandleRebuildIntellisenseNotification(
             }
         }
 
+        /// <summary>
+        /// Verify that the latest SqlParser (2016 as of this writing) is used by default
+        /// </summary>
+        [Test]
+        public void LatestSqlParserIsUsedByDefault()
+        {
+            // This should only parse correctly on SQL server 2016 or newer
+            const string sql2016Text =
+                @"CREATE SECURITY POLICY [FederatedSecurityPolicy]" + "\r\n" +
+                @"ADD FILTER PREDICATE [rls].[fn_securitypredicate]([CustomerId])" + "\r\n" +
+                @"ON [dbo].[Customer];";
+
+            // parse
+            var scriptFile = new ScriptFile();
+            scriptFile.SetFileContents(sql2016Text);
+            var langService = CreateLanguageService(scriptFile);
+
+            ScriptFileMarker[] fileMarkers = langService.GetSemanticMarkers(scriptFile).GetAwaiter().GetResult();
+
+            // verify that no errors are detected
+            Assert.AreEqual(0, fileMarkers.Length);
+        }
+
+        [Test]
+        [TestCase(
+            @"
+            SELECT * from sys.objects
+            --returning
+            select * s
+            ", 
+            false, 
+            "Should not detect non-T-SQL in a valid T-SQL script."
+        )]
+        [TestCase(
+            @"
+            SELECT * from [returning]
+            --returning
+            ", 
+            false, 
+            "Should not detect non-T-SQL in a valid T-SQL script, with a Non-T-SQL keyword as an object name"
+        )]
+        [TestCase(
+            "returning", 
```

Comment:
Can you expand this case so that it's an actual MySQL script instead of just a singular keyword?